src/libs/runtime-utils.js

```diff
@@ -325,7 +323,7 @@ export const appIsSettingUp = app => {
   return app && (app.status === 'PROVISIONING' || app.status === 'PRECREATING')
 }
 
-export const getCurrentPersistentDisk = (appType, apps, appDataDisks, workspaceName) => {
+export const getCurrentAppDataDisk = (appType, apps, appDataDisks, workspaceName) => {
```

Comment:
I think /** jsdoc */ style function comment blocks would go a long way to documenting these methods' intended use... me coming in cold, I have no idea why this one was renamed getCurrentAppDataDisk, and a different implementation is being put in for getCurrentPersistentDisk below.